api-lib/event_triggers/activity/activityHandler.ts

```diff
@@ -0,0 +1,142 @@
+import type { VercelRequest, VercelResponse } from '@vercel/node';
```

Comment:
now that you've created a folder named `activity`, having the files also be prefixed with `activity` seems redundant to me, and none of the other files have `Handler` in their name, so how about renaming to `index.ts` and `mutations.ts`?

Reply:
ok, addressed all the feedback.